docs/assets/css/site.css

```diff
@@ -0,0 +1,520 @@
+    background-color: transparent !important;
+}
+
+localize-button .inline-dropdown i,
+localize-button .btn.inline-dropdown.dropdown-toggle > span {
+    color: #A6A8AB;
+    border-bottom-color: #A6A8AB;
+    transition: all 0.3s ease-in-out;
+}
+
+localize-button .inline-dropdown:hover i,
+localize-button .btn.inline-dropdown.dropdown-toggle:hover > span {
+    color: #fff;
+    border-bottom-color: #fff;    
+}
+
+/*
+    Responsive Side Menu Button
+*/
+.side-nav-expand-btn {
+    background-color: #74E291;
+    box-shadow: 1px 1px 3px #000;
+}
+
+.side-nav-expand-btn:hover {
+    background-color: #5BC978;
+}
+
+@media (max-width: 992px) {
+    .sidebar {
+        background-color: #1b282e;
+    }
+}
+
+/*
+    Team Page
+*/
+.business-card .social .social-link .hpe-icon,
+.business-card .social .social-link .platform {
+    color: #E9E9E9;
+    opacity: 1;
+}
+
+.business-card .social .social-link:hover .hpe-icon,
+.business-card .social .social-link:hover .platform {
+    color: #fff;
+}
+
+/*
+    Page footer
+*/
+
+.footer-banner {
+    background-color: #171717;
+}
+
+.footer-banner .footer-title {
+    color: #b8ffd8;
+}
+
+.footer-banner .copyright,
+.footer-banner .feedback {
+    opacity: 0.5;
+    color: #b8ffd8;
+}
+
+/*
+    Blog Page
+*/
+.ebox-title,
+.ebox-content {
+    background-color: transparent;
+}
+
+.ebox-content {
+    border-color: #323d43;
+}
+
+.content-container .docs-content h1, 
+.content-container .docs-content h2, 
+.content-container .docs-content h3, 
+.content-container .docs-content h4, 
+.content-container .docs-content h5, 
+.content-container .docs-content h6
+{
+    font-family: Metric;
+    color: #fff;
+}
+
+.content-container .docs-content .no-posts {
+    margin-top: 20px;
+}
+
+.blog-post .blog-header {
+    padding-top: 0;
+}
+
+.blog-post .blog-header .blog-title h3 {
+    margin-top: 0;
+}
+
+/*
+    Code Snippets
+*/
+
+.highlighter-rouge pre.highlight {
+    background-color: #354247;
+    border-color: #192023;
+}
+
+code {
+    background-color: #354247;
+    border-color: #192023;
+    color: #5eb4ff;
+}
+
+.highlight .k,
+.highlight .p,
+.highlight .n {
+    color: #eee;
+}
+
+.highlight .m {
+    margin: 0;
+}
+
+.highlight .nt {
+    color: #5eb4ff;
+}
+
+.highlight .na {
+    color: #00caca;
+}
+
+.highlight .s {
+    color: #11dd48;
+}
+
+/*
+    Pagination Controls
+*/
+.pagination-container .btn-group .button-secondary {
+    background-color: #354247;
+    border-color: #192023;
+    box-shadow: none;
+    color: #E9E9E9;
+    transition: background-color 0.3s ease-in-out;
+}
+
+.pagination-container .btn-group .button-secondary.active {
+    background-color: #24524E;
+}
+
+.pagination-container .btn-group .button-secondary.active:hover {
+    background-color: #0B3935;
+}
+
+.pagination-container .btn-group .button-secondary:hover {
+    background-color: #1B282D;
+}
+
+/*
+    Team Page
+*/
+.flip-container .front,
+.flip-container .back {
+    background-color: #354247;
```

Comment:
Might be good to change the background color of the team member cards:

![image](https://user-images.githubusercontent.com/20795331/27142553-442d38d0-5123-11e7-84d1-b56af0a88ecc.png)

They are currently a bit blue which is quite different than any other colours on the site. Looking at the [Job service site](https://jobservice.github.io/job-service/pages/en-us/Team) which has a closer color scheme has darker card background colors which may be more suitable (or the colour of the site footer `#171717` also works pretty well), eg:

![image](https://user-images.githubusercontent.com/20795331/27142681-a9476f6a-5123-11e7-91a5-37fe67eaf286.png)


This colour could get carried across to the code snippets also, eg:

![image](https://user-images.githubusercontent.com/20795331/27142796-2287d0a4-5124-11e7-9397-fe7ab8c0ab0e.png)

-->

![image](https://user-images.githubusercontent.com/20795331/27142815-311c0810-5124-11e7-819e-3e9e620b30f5.png)